shared/Cargo.toml

```diff
@@ -23,8 +26,17 @@ rust_decimal_macros = "1.29.0"
 serde = { version = "1.0.158", features = ["derive"] }
 serde_json = "1.0.93"
 serde_with = "2.3.1"
-sqlx = { version = "0.6", features = [ "offline", "runtime-tokio-rustls", "postgres", "decimal", "uuid", "chrono", "json" ] }
-sqlxmq = { version = "0.4.1", default-features = false, features = [ "runtime-tokio-rustls" ] }
+sqlx = { version = "0.7.1", features = [
```

Comment:
can you format this so that everything is in a single line.

Reply:
done